src/visualization/Turi Create Visualization/src/VegaContainer.swift

```diff
@@ -278,6 +278,23 @@ class VegaContainer: NSObject, WKScriptMessageHandler {
         }
     }
     
+    public func send_proto(protobuf: String) {
```

Comment:
Is this used? It looks like elsewhere in the code, proto is only ever encoded as part of a JSON message.

Reply:
Updated code! Sorry forgot to push!

Reply:
No problem!